### ISSUE TITLE
Rechnungsworkflow: requester referenz

### ISSUE THREAD

**Issue by simoneOnFhir** [gematik/spec-E-Rechnung#11]
Aktuell wird der Requester nur als Telematik-ID referenziert.
* requester 1.. MS
  * identifier 1.. MS  
    * ^short = "Telematik-ID des Leistung
  * identifier only IdentifierTelematikId
Würde der Fachdienst den Task nicht einem registrierten Leistungserbringer-Benutzer (Practitioner?) zuordnen wollen, analog zur Auflösung der KV-Nummer eines Rechnungsempfängers zur Referenz auf einen Patienten/User?
Fehlt uns da noch ein Profil für Practitioner(Role)?

**Comment by simoneOnFhir** [comment]
Gleiche Frage auch für die Substitution von Invoice.issuer bei der Extraktion aus dem Bundle...

**Comment by PeterTabeling-gematik** [comment]
Ich denke, es wäre sinnvoll, hier die Daten eines Practitioner (?) aufzunehmen und nicht nur die Telematik-ID als Referenz zu haben. In jedem Fall bei der **Abfrage** der Rechnung als Datensatz. Dann sollte man nicht zusätzlich über die Telematik-ID die Daten des Erstellers der Rechnung abfragen müssen. "self-contained" fände ich besser weil einfacher für die Nutzer (speziell die Kostenträger) des Dienstes.
Spannend in dem Zusammenhang ist folgender Punkt:
Wir haben einerseits die Identität des Nutzers (Institution), der den Rechnungsdatensatz in den Fachdienst geschrieben hat - sozusagen der technische Rechnungsersteller. Hier könnte man die Daten aus dem Nutzerkonto (über die Telematik-ID des angemeldeten Nutzers verknüpft) übernehmen.
Andererseits könnte es ja sein, dass dieser Nutzer ein Abrechnungsdienstleister ist, der rechtlich aber nicht als Rechnungsaussteller gilt - wenn er nämlich im Auftrag der Arztpraxis die Rechnung erstellt, die eigentlich die Forderung erhebt. Deren Daten müssten wir in jedem Fall schon bei der Anlage der Rechnung mit übergeben, denn es ist nicht gesagt, dass die Arztpraxis einen Nutzer-Account im Fachdienst hat. Und man würde diese Daten ja mit signieren wollen.

Beantwortet das die Frage? :-)

**Comment by PeterTabeling-gematik** [comment]
sonst besser mal Telefonieren/MS Teams. Im nächsten FHIR Workshop oder auch gerne vorher ...

**Comment by simoneOnFhir** [comment]
Ok, @alexzautke dann bräuchten wir wohl ein Practitioner-(evtl. auch Practitioner-Role?)Profil, das die Benutzerdaten eines Arztes enthält. Der Server muss dann bei der Entgegennahme eines Bundles die Logische Referenz der Telematik-ID mit der Referenz auf die entsprechende Practitioner-Ressource substituieren (oder besser den Benutzer-Kontext aus dem Auth-Token verwenden?)

Soll die Rechnung abgewiesen werden, wenn die Telematik-ID des Benutzers nicht mir der in der ID in Invoice.issuer übereinstimmt @PeterTabeling-gematik ?

**Comment by alexzautke** [comment]
Sollten wir nicht schauen wie das potentiell mit dem gematik VZD zu lösen ist? Nochmal Practitioner/PractitionerRole-Daten zu pflegen klingt wenig sinnvoll für mich.

**Comment by SvenSommer** [comment]
@alexzautke Ja, bitte den FHIR-VZD als zentrale Instanz für die die Verwaltung und Pflege von Practitioner, und Organisation Ressourcen (Arzt und Apotheker) vorsehen. Lasst uns dazu gerne mal kurz brainstormen. :)

**Comment by PeterTabeling-gematik** [comment]
> Ok, @alexzautke dann bräuchten wir wohl ein Practitioner-(evtl. auch Practitioner-Role?)Profil, das die Benutzerdaten eines Arztes enthält. Der Server muss dann bei der Entgegennahme eines Bundles die Logische Referenz der Telematik-ID mit der Referenz auf die entsprechende Practitioner-Ressource substituieren (oder besser den Benutzer-Kontext aus dem Auth-Token verwenden?)
> 
> Soll die Rechnung abgewiesen werden, wenn die Telematik-ID des Benutzers nicht mir der in der ID in Invoice.issuer übereinstimmt @PeterTabeling-gematik ?

Hallo @simoneOnFhir, 

ich denke, wir sollten bedenken, dass ein Abrechnungsdienstleister unter Umständen "nur im Auftrag handelt" in dem Sinne, dass er zwar - technisch - die Rechnungen erstellt, aber der rechtliche Rechnungssteller (Inhaber der Forderung) könnte die Arztpraxis bleiben. Dann wäre die Arztpraxis (und damit auch alle benötigten Angaben dazu) in der Rechnung als "Issuer" aufzuführen. Aber der Nutzer, der technische die Rechnung in den Fachdienst eingebringt, wäre der Abrechnungsdienstleister.

--> Können wir bei dieser Konstellation überhaupt erwarten (verlangen?), dass die Arztpraxis mit einer Telematik-ID ausgestattet ist? Wie wäre das bei einer rein privat abrechnenden Arztpraxis, die alles über einen ADL abwickelt, so wie oben skizziert? Letzterer würde eine Telematik-ID haben (haben müssen), denn nur er müsste an die TI angeschlossen sein und sich als Institution per SMC-B ausweisen müssen.

LG Peter

**Comment by PeterTabeling-gematik** [comment]
P.S. den FHIR VZD haben wir bislang nicht vorgesehen in der ersten Ausbaustufe, da ja alle, die sich am Fachdienst registrieren, dort mit einem Nutzerkonto inkl. den wichtigsten Angaben, angelegt wären (ausgehend von den Infos im ID Token, die sich aus der SMC-B ergeben) - auch Rechnungsersteller und Kostenträger. Daher wollten wir zunächst auf diese zusätzliche Abhängigkeit (vom VZD) verzichten. Perspektivisch macht es aber natürlich Sinn ...

**Comment by simoneOnFhir** [comment]
Wie kann eine rein private Arztpraxis ohne TelematikID und ohne Benutzeraccount im Fachdienst referenziert bzw. identifiziert werden? 
Das würde nur funktionieren, wenn wir deren komplette Stammdaten jedes mal mitschicken und wir hätten keine Möglichkeiten der Validitätsprüfung.

**Comment by PeterTabeling-gematik** [comment]
japp. das wäre wohl so ... :-/
Wobei wir (bzw. der Ersteller der Rechnung) uns auf die Daten beschränken könnten, die für eine Rechnung erforderlich sind.

**Comment by simoneOnFhir** [comment]
Nochmal zum mitdenken. Es kann RechnungsERSTELLER geben, die keine Telematik-ID haben und keinen Fachdienstaccount (sprich: "jeder"??). Die einzige Kontrollinstanz in diesem Fall wäre der Abrechnungsdienstleister, der uns sein Indianer-Ehrenwort gibt, dass er nur Rechnungen für "echte" Leistungeerbringer erstellt??


Sind die folgenden Annahmen korrekt?
"wenn Einreicher gleich Aussteller, dann Telematik-ID immer vorhanden" 
"wenn Einreicher ungleich Aussteller, dann Telematik-ID für Einreicher immer vorhanden, für Aussteller nicht immer"
"wenn der Aussteller über eine TelematikID verfügt, dann ist diese dem Einreicher bekannt und kann stets übermittelt werden"
"Leistungserbringer/Aussteller ohne TelematikID können keinen eigenen Fachdienstbenutzer anlegen"
"Leistungerbringer/Aussteller ohne TelematikID können Rechnungen NUR über einen Dienstleister einreichen"
"Rechnungen von Leistungerbringern/Ausstellern ohne TelematikID haben zwangläufig einen abweichenden Einreicher"

**Comment by patrick-werner** [comment]
fyi: bei der epaMedikation setzen wir voll auf das FHIR VZD, für Organization, PractitionerRole und Practitioner
